internal/cmd/stack/run_replan.go

```diff
@@ -0,0 +1,141 @@
+package stack
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/manifoldco/promptui"
+	"github.com/shurcooL/graphql"
+	"github.com/urfave/cli/v2"
+
+	"github.com/spacelift-io/spacectl/internal/cmd/authenticated"
+)
+
+func runReplan(cliCtx *cli.Context) error {
+	stackID, err := getStackID(cliCtx)
+	if err != nil {
+		return err
+	}
+
+	runID := cliCtx.String(flagRequiredRun.Name)
+
+	var resources []string
+	if cliCtx.Bool(flagInteractive.Name) {
+		var err error
+		resources, err = interactiveResourceSelection(cliCtx, stackID, runID)
+		if err != nil {
+			return err
+		}
+	} else {
+		resources = cliCtx.StringSlice(flagResources.Name)
+	}
+
+	if len(resources) == 0 {
+		return fmt.Errorf("no resources targeted for replanning: at least one resource must be specified")
+	}
+
+	var mutation struct {
+		RunTargetedReplan struct {
+			ID string `graphql:"id"`
+		} `graphql:"runTargetedReplan(stack: $stack, run: $run, targets: $targets)"`
+	}
+
+	targets := make([]graphql.String, len(resources))
+	for i, resource := range resources {
+		targets[i] = graphql.String(resource)
+	}
+
+	variables := map[string]interface{}{
+		"stack":   graphql.ID(stackID),
+		"run":     graphql.ID(runID),
+		"targets": targets,
+	}
+
+	if err := authenticated.Client.Mutate(cliCtx.Context, &mutation, variables); err != nil {
+		return err
+	}
+
+	fmt.Printf("Run ID %q has is being replanned\n", runID)
+	fmt.Println("The live run can be visited at", authenticated.Client.URL(
+		"/stack/%s/run/%s",
+		stackID,
+		mutation.RunTargetedReplan.ID,
+	))
+
+	if !cliCtx.Bool(flagTail.Name) {
+		return nil
+	}
+
+	terminal, err := runLogsWithAction(cliCtx.Context, stackID, mutation.RunTargetedReplan.ID, nil)
+	if err != nil {
+		return err
+	}
+
+	return terminal.Error()
+}
+
+func interactiveResourceSelection(cliCtx *cli.Context, stackID, runID string) ([]string, error) {
+	resources, err := getRunChanges(cliCtx, stackID, runID)
+	if err != nil {
+		return nil, err
+	}
+
+	templates := &promptui.SelectTemplates{
+		Label:    "{{ . }}?",
+		Active:   "\U0001F680 {{ .Address | cyan }} \U0001F680",
```

Comment:
What does `\U0001F680` mean?

Reply:
Rocket emoji 🚀

Reply:
It would be nice to replace those with a const. Maybe?

```
Active: rocketEmoji + " {{ .Address | cyan }} " + rocketEmoji
```